src/qt/editor/toolsmenu.cpp

```diff
@@ -80,7 +80,7 @@ void createProcessorDocMenu(InviwoApplication *app, QMenu *docsMenu) {
                 const auto processor = factory->create(pfo->getClassIdentifier());
                 const auto &inports = processor->getInports();
                 const auto &outports = processor->getOutports();
-                const auto &properties = processor->getProperties();
```

Comment:
Well, this will include all properties, but not the hierarchy. eg:
* Raycasting properties
* TF
* SamplingRate
* ...
* Lighting Property
* Ambiend
* Diffuse 
* ... 

### where one might want something like this

* Raycasting properties
* * TF
* * SamplingRate
* * ...
* Lighting Property
* * Ambiend
* * Diffuse 
* * ...

Reply:
So far, the indentation is hardly used (except for VolumeSlice) for composites. But I've stumbled upon this ('noiseprocessor.h') where a second level is used to indicate options of an optionproperty (and that pattern appears several times):
```c++
/**
 * ### Properties
 *   * __size__ Size of the output image.
 *   * __type__ Witch type of noise to generate.
 *   * __range__ The min/max values of the output values (default: [0 1]).
 *   * __Perlin Noise:__
 *     + __persistence__ Controls the sharpnes in Perlin noise
 *     + __levels__ Numbers of levels used in the generation.
 *   * __Poisson Disk Sampling:__
 *     + __poissonDotsAlongX__ Average number of points along the x-axis.
 *     + __poissonMaxPoints__ Maximum number of output points (total).
 */
```